File: SIM/model/biometric.py
from model import Base


class BiometricSensor(Base):

    def __init__(self, code="", token=""):
        
        super(BiometricSensor, self).__init__(code)
        self.token = token

    def __repr__(self):

        return f"[ BIOMETRIC SENSOR ] -> " \
               f"[ ID ]: {self.id}" \
               f"[ TOKEN ]: {self.token}" \
               f"[ TIME ]: {self.data}"
    
    

File: gui/src/components/status/Biometric.js
import React from "react";

class Biometric extends React.Component {

    constructor(props) {

        super(props);
        this.state = { biometric: props.data };

    }

    render() {
        return (
            <div className="row border">

                <div className="row"><b>Biometric Sensor</b></div>
                <div className="row">
                    <div className="col">
                        Token:&nbsp;
                        {this.state.biometric.token}
                    </div>
                </div>

            </div>
        )
    }

}

export default Biometric;